### PR TITLE
Update group.md

### PR DESCRIPTION
**Pull request by henrymbuguakiarie**
-------
cc: @psaffaie

### CODE REVIEW

**Comment by Jordanndahl** [comment]
@henrymbuguakiarie the changes are good to go! Thanks!

**Comment by henrymbuguakiarie** [comment]
#sign-off

File: api-reference/v1.0/resources/group.md [hunk 1]
```diff
@@ -162,8 +162,8 @@ This resource supports:
...
-| HiddenMembership | Owner permission is needed to join the group.<br>Non-members cannot view the attributes of the group.<br>Non-members cannot see the members of the group.<br>Administrators (global, company, user, and helpdesk) can view the membership of the group.<br>The group appears in the global address book (GAL). |
+| Private          | Owner permission is needed to join the group.<br>Anyone can view the attributes of the group.<br>Anyone can see the members of the group.                                                                                                                                                              |
+| HiddenMembership | Owner permission is needed to join the group.<br>Guests cannot view the attributes of the group.<br>Non-members cannot see the members of the group.<br>Administrators (global, company, user, and helpdesk) can view the membership of the group.<br>The group appears in the global address book (GAL). |
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/group.md:166]: nit
```suggestion
| HiddenMembership | Owner permission is needed to join the group.<br>Guest users cannot view the attributes of the group.<br>Non-members cannot see the members of the group.<br>Administrators (global, company, user, and helpdesk) can view the membership of the group.<br>The group appears in the global address book (GAL). |
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @henrymbuguakiarie for making this clarification. Please make the same changes to the beta doc as well.

Approved with suggestions.

**Pull request merged**